modules/webbrowser/renderhandlergl.h

```diff
@@ -0,0 +1,78 @@
+/*********************************************************************************
+ *
+ * Inviwo - Interactive Visualization Workshop
+ *
+ * Copyright (c) 2018 Inviwo Foundation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *********************************************************************************/
+
+#ifndef IVW_RENDERHANDLERGL_H
+#define IVW_RENDERHANDLERGL_H
+
+#include <modules/webbrowser/webbrowsermoduledefine.h>
+#include <modules/opengl/texture/texture2d.h>
+
+#include <warn/push>
+#include <warn/ignore/all>
+#include <include/cef_render_handler.h>
+#include <warn/pop>
+
+namespace inviwo {
+
+/** \class RenderHandlerGL
+ * Copies web page into a Texture2D each time it has been painted by the browser and calls
+ * onWebPageCopiedCallback afterwards.
+ */
+class RenderHandlerGL : public CefRenderHandler {
+public:
+    RenderHandlerGL(std::function<void()> onWebPageCopiedCallback);
+    void updateCanvasSize(size2_t newSize);
+
+    bool GetViewRect(CefRefPtr<CefBrowser> browser, CefRect &rect) override;
```

Comment:
any reason for uppercase function names?

Reply:
Cef standard (overriding their function)